test/corpus/expressions.txt

```diff
@@ -982,7 +998,7 @@ method call with keyword args
 foo(a: true)
 foo a: true
 foo B: true
-foo a:
+foo a: ;
```

Comment:
was this semicolon intended here?

Reply:
Yes it is intended. Without the semicolon the test case was not valid Ruby code.